src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractCommon.4.1.0.cs

```diff
@@ -15,7 +15,6 @@ namespace MessageContractCommon
     public class MessageContractConstants
     {
         // CONSTANTS
-        public const string wrapperName = "CustomWrapperName";
```

Comment:
I'd prefer not to have a const defined in the common code for wrapper name as it could be different per test, although it happens to be the same for the existing 3 tests.